cirq/study/trial_result.py

```diff
@@ -294,3 +295,64 @@ def __add__(self, other: 'cirq.TrialResult') -> 'cirq.TrialResult':
                                               other.measurements[key],
                                               axis=0)
         return TrialResult(params=self.params, measurements=all_measurements)
+
+    def _json_dict_(self):
+        packed_measurements = {}
```

Comment:
factor this out into its own class that supports the json protocol

Reply:
Talked offline and decided it's unnecessary.